app/config/local.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  storybooksEnabled: true,
+}
```

Comment:
This should be in the gitignote*, this is Add Jam convention so not all that important in this context but why we do this is so local allows team members to put their own config in (apiUrl/storybook enabled) and it won't cause conflicts/impact other team members.

* my bad , ive not added it but please add it now.

Reply:
hmm, you have added it, it seems?

`# Local config
app/config/local.js`